web/src/components/map/MarkerButton/IconButton.ts

```diff
@@ -25,15 +25,24 @@ function MarkerButton(markerImage, markerType, markerCaption) {
   });
 }
 
+function makeImageUrl(image) {
+
```

Comment:
I implemented this in https://github.com/helpbuttons/helpbuttons/blob/main/web/src/shared/sys.helper.tsx#L26 please review your changes according to this change :rocket:

Reply:
I'll revert the request yes